Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -287,7 +287,22 @@ spec:
     * This format is required in order to use the NetworkAttachmentDefinition across namespaces.
     * In Openshift, to use a NetworkAttachmentDefinition (NAD) across namespaces, the NAD must be deployed in the `default` namespace. The NAD is then referenced with the namespace: `default/rook-public-nw`
 
-#### Known limitations with Multus
+##### Validating Multus configuration
+
+We **highly** recommend validating your Multus configuration before you install Rook. A tool exists
+to facilitate validating the Multus configuration. After installing the Rook operator and before
+installing any Custom Resources, run the tool from the operator pod.
+
+The tool's CLI is designed to be as helpful as possible. Get help text for the multus validation
+tool like so:
+```console
+kubectl --namespace rook-ceph exec -it deploy/rook-ceph-operator -- rook ctl multus validation run --help
```

Comment:
Another approach to consider is to create a sample job manifest that could run the job, similar to our osd-purge.yaml, but perhaps as a separate PR if it makes sense.

Reply:
I'll create a follow-up issue for this. It seems a good way to make sure logs are preserved if the pod running the tool fails.